test/testinfra.py

```diff
@@ -193,6 +175,22 @@ def __init__(self, base="/repos/cockpit-project/cockpit/"):
                 raise
         self.available = self.token and True or False
 
+        # Try to load the whitelists
```

Comment:
To be fair, the token code has about half the number of lines. But ok, I can live with this. Sometimes it's easier to debug if you don't have to wade through lots of code to see what's actually being done. `read_whitelist` is very concise.